src/Utils/Json.php

```diff
@@ -89,7 +89,7 @@ public static function decode($json, $options = 0)
 		}
 		$value = call_user_func_array('json_decode', $args);
 
-		if ($value === NULL && $json !== '' && strcasecmp($json, 'null')) { // '' is not clearing json_last_error
+		if ($value === NULL && $json !== '' && strcasecmp(trim($json, " \t\n\r"), 'null') != 0) { // '' is not clearing json_last_error
```

Comment:
Considering larger input, wouldn't `preg_match` be a better choice performance-wise?
```php
!preg_match('~^\s*null\s*$~is', $json)
```

http://3v4l.org/X9X9u